slides/src/content/components.md

```diff
@@ -312,6 +312,63 @@ class ChildComponent {}
 
 ---
 
+## Defining multiple projection areas
+
+- Multiple `<ng-content>` tags can be used in a template by using the `select` attribute
+- In the template, we can use an HTML tag, say, `<header>` to specify the position of projected content to the `ng-content` with `select="header"`
+
+```ts
+  // app.component.ts
+  template: `<rio-child>
+              <header>
+               <p>This is my header content</p>
+              </header>
+              <footer>
+                <p>This is my footer content</p>
+              </footer>
+             </rio-child>`
+```
+
+```ts
+  // child.component.ts
+  template: `<h4>Child Component</h4>
+             <ng-content select="header"></ng-content>
+             <ng-content select="footer"></ng-content>`
+```
+
+[View Example](https://plnkr.co/edit/kwD3iKLU8mELAoHj2fBv?p=preview)
```

Comment:
The plunkers are using old angular deps (angular 2.1.0, rxjs beta, etc.) could you upgrade the dependencies of both? Here's an example: https://plnkr.co/edit/ufRNSFV9EYeBmyItaF6Q?p=info